doc/iterations/2023.6/shapes/sendMessageToFrontend.adoc

```diff
@@ -0,0 +1,59 @@
+= (L) Display feedback message on the frontend
+
+== Problem
+
+It's not possible for a specifier to alert end user after a tool action.
+
+== Key Result
+
+The specifier can call an API in his java service to display a message on the UI.
+The representation of the messages can be specified to alert the end user.
+
+== Solution
+
+Offer an API to display messages on the UI.
+
+[source,java]
+----
+interface IFeedbackMessageService {
+    void addFeedbackMessage(String message, FeedbackLevel level);
+    void addFeedbackMessage(String message, FeedbackStyle style);
+}
+----
+with
+[source,java]
+----
+enum FeedbackLevel {
+    WARNING,
+    INFO,
+    ERROR
+}
```

Comment:
If we keep an enum, the values should probably be sorted so that INFO < WARNING < ERROR.

Reply:
I changed the order